### ISSUE TITLE
Set up continuous fuzzing

### ISSUE THREAD

**Issue by dominikh** [dominikh/go-tools#569]
Our printf parser has fuzzing, our upcoming pattern parser will have fuzzing, and if we integrate gosmith, we could fuzz staticcheck itself, too (gosmith would also be beneficial for fuzzing the matcher component of patterns). 

Seeing how continuous fuzzing services are up and coming and are free to use for open source projects, we should probably set it up and benefit from the free CPU time.

**Comment by ksurent** [comment]
Had [this](https://fuzzit.dev) in my bookmarks. They claim it's free for OSS (at the bottom of the index page).

PS: I am not affiliated with them or even have tried it for myself.

**Comment by ksurent** [comment]
Looks like you can also qualify for their reward program [here](https://fuzzit.dev/2019/08/12/announcing-rewards-for-go-rust-oss-projects/).

**Comment by dominikh** [comment]
Yes, I'll likely use fuzzit.dev – They're already a sponsor of staticcheck, so I probably won't pester them about the reward program ;)
